lib/project_types/theme/themekit.rb

```diff
@@ -5,19 +5,29 @@ class Themekit
     class << self
       def create(ctx, password:, store:, name:)
         stat = ctx.system(THEMEKIT,
-                          "new",
+                          'new',
                           "--password=#{password}",
                           "--store=#{store}",
                           "--name=#{name}")
         stat.success?
       end
 
+      def deploy(ctx)
+        unless push(ctx, files: nil, flags: nil, remove: false)
```

Comment:
There's no need to set the named variables if they're using the default values, can we just remove them from the call?